public/scripts/templates/migrateInstructPrompt.html

```diff
@@ -0,0 +1,7 @@
+<h3>
+    This instruct template also contains a system prompt.
+</h3>
+<div class="marginBot10">
+    Would you like to migrate the system prompt from the template?
```

Comment:
Maybe we should also say that otherwise the specific system prompt from the instruct template will not be used, and the current system prompt stays active.

I mean, should be obvious. But ya know...